### ISSUE TITLE
cleaning up stale branches...

### ISSUE THREAD

**Issue by ryanpeek** [Delta-Stewardship-Council/swg-21-connectivity#156]
@goertler @layamane @shrkhanna @shrkhanna @catarfish @dylanchapple1 @matt-berg @estumpnerUSGS 

Hi folks! 👋 
I'm taking some time today and tomorrow to look through our repo and make sure I understand where/how all the pieces/code/data came to be, clean up anything that we know how/why it is there (will ask first and document thoroughly before doing anything), etc. I will revise and update the workflow diagram as well.

One thing that would be helpful is if you can look through the branches...many are already merged. If the branch is not active or needed anymore (most!), please delete the branch here, and probably delete locally (`git branch -D BRANCH_NAME_TO_DELETE`). If you aren't sure, please let me know so we can review and clean up together. Ideally all stale/merged branches should be trimmed away, so it's clear where active work is happening.

Thanks team! 🥳

**Comment by goertler** [comment]
Thanks for working through our repo @ryanpeek ! Most of those stale branches were mine, and I've deleted all but prediction_plots since I may be changing that plot for @shrkhanna in the near future.

**Comment by estumpnerUSGS** [comment]
Hi @ryanpeek - I believe I have only one fork and no stale branches but could use some reassurance from reviewing it together.

**Comment by ryanpeek** [comment]
yup it looks good by me...I have an open branch with some edits (but waiting to tidy up from all first), and Pascale just merged a branch, so overall I think we are good!
